SECURITY.md

```diff
@@ -0,0 +1,83 @@
+## Configuring Security
+
+
+The following variables configure container security using the socialLogin-1.0 feature.  
+
+### Configuration needed at image build time:
+
+ * The environment variable `sec_sso_providers` must be defined and contain a space delimited list of the identity providers to use. If more than one is specified, the user will be able to choose which one to authenticate with. Valid values are any of `oidc oauth facebook twitter github google linkedin`.  Specify `ARG sec_sso_providers="(your choice goes here)"` in your Dockerfile.
+
+ * Providers usually require the use of HTTPS.  Specify `ARG TLS=true` in your Dockerfile. 
+
+ * To automatically trust certificates from well known identity providers, specify  `ARG SEC_TLS_TRUSTDEFAULTCERTS=true` in your Dockerfile.
+
+### Configuration needed at image build time or at container deploy time:
+
+Each provider needs some additional configuration to be functional -  a client Id, client secret and sometimes more. These variables can be supplied in several ways:
+  * At build time, they can be variables in a server.xml file (`<variable name="foo" value="bar" />`).
+  * At build time, they can be ENV variables in the Dockerfile, this is less secure (`ENV name=value`).
+  * They can be passed as environment variables to the Docker container when it is deployed. 
+  * They can be supplied in a deployment yaml file or by the Liberty operator, which will pass them to the container at deploy time.
```

Comment:
can we link here to the Open Liberty Operator doc so that users know how to use it?

Reply:
I think it would be good to have a sample `Dockerfile` for the application image that demonstrated using these env vars (and calling configure.sh), paired with a sample Open Liberty Operator YAML file - that way the users can read about this config but can also have something they can cut / paste into their repo.

Reply:
fixed,  I'll need to get the yaml from Leo.

Reply:
I'll add the operator yaml via a separate PR.